schedule/yast/lvm/lvm+cancel_existing_cryptlvm_pvm.yaml

```diff
@@ -3,9 +3,6 @@ description: >
   Test suite cancels encrypted partitions activation and performs installation
```

Comment:
we need to unify this file too, otherwise we cannot create new one in test_cases.

Reply:
In this case ppc64le tests require `textmode` desktop in order to work.
I talked with the team about the possibility to swap to `gnome` desktop but it seems that Gnome desktop requires at least 4GB of RAM and the defined machine `spvm` use LPAR which doesn't allow to increase the RAM size by vars.
They suggest me to swap machines to `hmc`, giving a short test it doesn't work easily so I ended up at the only way I found to make all architectures to work. Maybe I can touch the default YAML files but it seems to be pretty breaker at multiple points to me.